README.md

```diff
@@ -147,6 +147,14 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Cobertura](http://cobertura.github.io/cobertura/) - Relies on offline (or static) bytecode instrumentation and class loading to collect code coverage metrics; GPLv2 licensed.
 * [JaCoCo](http://eclemma.org/jacoco/) - Framework that enables collection of code coverage metrics, using both offline and runtime bytecode instrumentation; prominently used by EclEmma, the Eclipse code-coverage plugin.
 
+## Code Generators
+
+*Code that generates bug-prone boilerplate code.*
```

Comment:
To be clear, I was not talking about the code that the code generators
generate. "bug-prone" was the code you would otherwise have to write
yourself, by hand.

On Fri, Jul 1, 2016 at 4:05 AM, Andreas Kull <notifications@github.com>
wrote:

> In README.md
> <https://github.com/akullpp/awesome-java/pull/375#discussion_r69263432>:
>
> > @@ -147,6 +147,14 @@ A curated list of awesome Java frameworks, libraries and software.
> >  * [Cobertura](http://cobertura.github.io/cobertura/) - Relies on offline (or static) bytecode instrumentation and class loading to collect code coverage metrics; GPLv2 licensed.
> >  * [JaCoCo](http://eclemma.org/jacoco/) - Framework that enables collection of code coverage metrics, using both offline and runtime bytecode instrumentation; prominently used by EclEmma, the Eclipse code-coverage plugin.
> >
> > +## Code Generators
> > +
> > +*Code that generates bug-prone boilerplate code.*
>
> bug-prone? That is rather harsh and opinionated.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/akullpp/awesome-java/pull/375/files/a4eafb68cbac8895c1c4467fc5f862cb96effd1e#r69263432>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANWZd3N801R7WS1YP2bdG1sYPYd5gwuks5qRMpfgaJpZM4I_Khm>
> .
>